src/turing_MCMC.jl

```diff
@@ -0,0 +1,49 @@
+using Turing
+
+function pinn(prob::DiffEqBase.DEProblem, parameters, dataset, recon)
+    chain = recon(parameters)
+
+    # output PINN predictions
+    sol
+end
+
+myloglikelihood(x, μ) = loglikelihood(MvNormal(μ, 1), x)
+
+# priors: pdf for W,b + pdf for ODE params
+function bayesian_pinn_ode(prob::DiffEqBase.DEProblem, chain, priors, dataset, ts;
+                           sampling_strategy = NUTS(0.65), num_samples = 1000,
+                           syms = [Turing.@varname(theta[i]) for i in 1:length(priors)])
+    param_initial, recon = Flux.destructure(chain)
+    nparameters = length(param_initial)
+
+    alpha = 0.09
+    sig = sqrt(1.0 / alpha)
+    @model function bayes_pinn(dataset)
+        theta = Vector{T}{undef, length(priors)}
+        for i in eachindex(priors)
+            theta[i] ~ NamedDist(priors[i], sym[i])
+        end
+        nnparameters ~ MvNormal(zeros(nparameters), sig .* ones(nparameters))
+
+        preds = pinn(prob, parameters, dataset, recon)
```

Comment:
```suggestion
        preds = pinn(prob, nnparameters, dataset, recon)
```
Did you mean this here?